server/src/middleware/validate_token.rs

```diff
@@ -27,18 +25,15 @@ pub async fn validate_token(
             let res = match known_token.status {
                 TokenValidationStatus::Validated => match known_token.token_type {
                     Some(TokenType::Frontend) => {
-                        trace!("Got FE token validated {:?}", known_token);
                         if req.path().contains("/api/frontend") || req.path().contains("/api/proxy")
                         {
-                            trace!("Was allowed to access");
                             srv.call(req).await?.map_into_left_body()
                         } else {
                             req.into_response(HttpResponse::Forbidden().finish())
                                 .map_into_right_body()
                         }
                     }
                     Some(TokenType::Client) => {
-                        trace!("Got Client token validated {:?}", known_token);
```

Comment:
Any reason why we're getting rid of these now?

Reply:
We've verified that it triggers, and we have the /internal-backstage/tokens endpoint to see which tokens we know about, so cleaned it up.